README.md

```diff
@@ -1,9 +1,71 @@
-# bc-parks-adventure-mobile
+# Parks Adventure Mobile
 
-A mobile app to help users find and explore BC Parks.
+Find and explore the wonders of **BC Parks**.
+
+## Features
+
+Two of the main screens, `Explore` and `Find`, provide different ways to
+discover parks. Once a user finds a park they can find more information about
+that park by navigating to the `Details` page. Then if that park is of interest
+the `Favourites` page will help keep it close at hand. Users can also stay
+up-to-date on the access status of parks (open, close, etc.) and any alert or
+advisories they should be aware of before traveling.
+
+---
+
+### 1. Explore
+
+<div style="display:flex;align-items:center;">
+  <kbd><img src="./assets/ExplorePage.PNG" width="175" /></kbd>
+  <p style="width: 400px;margin-left: 50px;">The <code>Explore</code> page shows
+  users nearby parks which offer <strong>popular activities and facilities</strong>.</p>
+</div>
+
+---
+
+### 2. Favourites
+
+<div style="display:flex;align-items:center;">
+  <kbd><img src="./assets/FavouritesPage.PNG" width="175" /></kbd>
```

Comment:
**suggestion (non-blocking)**: Make all extensions lowercase for files and `src` attributes.

Some of these extensions are capitalized and some aren't, and the actual files are sometimes capitalized and sometimes not. This works on MacOS as their filesystem is case-insensitive but results in some broken image references on the web.

Reply:
Oops. I thought I had changed all the file extensions. Thanks for catching this.